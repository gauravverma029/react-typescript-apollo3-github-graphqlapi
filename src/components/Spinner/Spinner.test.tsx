import React from "react";
import { cleanup, render } from "@testing-library/react";

import Spinner from "./index";

afterEach(cleanup);

describe("Spinner Component inital Loading when Content not present", () => {
  test("Should Render", async () => {
    const { asFragment } = render(<Spinner />);
    const html = asFragment();
    expect(html).toMatchSnapshot();
  });
});
