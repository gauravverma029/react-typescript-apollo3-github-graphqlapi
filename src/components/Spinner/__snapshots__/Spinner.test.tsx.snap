// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component inital Loading when Content not present Should Render 1`] = `
<DocumentFragment>
  <div
    class="w-9/12 mx-auto border border-gray-300 shadow rounded-md p-4"
    data-testid="loading"
  >
    <div
      class="animate-pulse flex space-x-4"
    >
      <div
        class="flex-1 space-y-4 py-1"
      >
        <div
          class="h-4 bg-gray-400 rounded w-3/4"
        />
        <div
          class="space-y-2"
        >
          <div
            class="h-4 bg-gray-400 rounded"
          />
          <div
            class="h-4 bg-gray-400 rounded w-5/6"
          />
        </div>
      </div>
    </div>
    <div
      class="animate-pulse flex space-x-4"
    >
      <div
        class="flex-1 space-y-4 py-1"
      >
        <div
          class="h-4 bg-gray-400 rounded w-3/4"
        />
        <div
          class="space-y-2"
        >
          <div
            class="h-4 bg-gray-400 rounded"
          />
          <div
            class="h-4 bg-gray-400 rounded w-5/6"
          />
        </div>
      </div>
    </div>
    <div
      class="animate-pulse flex space-x-4"
    >
      <div
        class="flex-1 space-y-4 py-1"
      >
        <div
          class="h-4 bg-gray-400 rounded w-3/4"
        />
        <div
          class="space-y-2"
        >
          <div
            class="h-4 bg-gray-400 rounded"
          />
          <div
            class="h-4 bg-gray-400 rounded w-5/6"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
